deepparse/app/app.py

```diff
@@ -33,7 +34,7 @@ async def lifespan(application: FastAPI):  # pylint: disable=unused-argument
     logger.debug("Downloading models")
     download_models()
     for model in MODEL_MAPPING_CHOICES:
-        if model not in ["fasttext", "fasttext-attention", "fasttext-light"]:
+        if model not in ["fasttext", "fasttext-attention", "fasttext-light"]:  # Skip fasttext models
```

Comment:
Reason to skip?

Reply:
Thought I answered this one, sorry. No reason to skip when the PR is ready to merge.